theming/theme-package.rst

```diff
@@ -479,22 +476,22 @@ As a starting point we use this rules set:
      </replace>
 
      <!-- Left column -->
-     <rules if-content="//*[@id='portal-column-one']">
+     <rules css:if-content="#portal-column-one">
        <replace css:theme="#column1-container">
-           <div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="sidebar">
+           <div class="col-xs-6 col-sm-3 sidebar-offcanvas">
```

Comment:
Got it!


On Mon, Nov 23, 2015 at 4:06 PM Davi Lima <notifications@github.com> wrote:

> In theming/theme-package.rst
> <https://github.com/plone/training/pull/177#discussion_r45680523>:
>
> >         <replace css:theme="#column1-container">
> > -           <div class="col-xs-6 col-sm-3 sidebar-offcanvas" id="sidebar">
> > +           <div class="col-xs-6 col-sm-3 sidebar-offcanvas">
>
> As I mentioned here
> <https://github.com/plone/training/issues/174#issuecomment-159054729>,
> because of plone/plonetheme.barceloneta#53
> <https://github.com/plone/plonetheme.barceloneta/pull/53>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plone/training/pull/177/files#r45680523>.
>